### DISCUSSION TITLE
Aggiungere dati storici al pannello energia

### DISCUSSION THREAD

**Discussion by faustozago** [indomus/forum#2643] [HUB: Home Assistant]
Ciao a tutti,
questo mese mi ha abbandonato il mio sensore energia POWR320, che3 ho sostituito dopo una settimana con uno uguale.
Il problema è che ho perso tutti i dati registrati.
C'è un modo per inserire questi dati (non dico giornalmente , ma un totale mensile il primo del mese? (i totali mensili li ho tutti trascritti in altro documento).

Ho gia modificati i valori esistenti da STRUMENTI SVILUPPATORI/STATISTICHE/REGOLA LA SOMMA ,  ma se in un giorno non ci sono misurazioni, non posso regolare/inserire niente.....

**Comment by faustozago** [comment]
Qualcuno sa dove vengono salvati i dati del pannello energia , oppure dei vari sensori ?
Se è un file comprensibile potrei tentare di aggiungere li qualche voce.....

**Comment by KC-inDomus** [comment]
> Ciao a tutti, questo mese mi ha abbandonato il mio sensore energia POWR320, che3 ho sostituito dopo una settimana con uno uguale. Il problema è che ho perso tutti i dati registrati. C'è un modo per inserire questi dati (non dico giornalmente , ma un totale mensile il primo del mese? (i totali mensili li ho tutti trascritti in altro documento).
> 
> Ho gia modificati i valori esistenti da STRUMENTI SVILUPPATORI/STATISTICHE/REGOLA LA SOMMA , ma se in un giorno non ci sono misurazioni, non posso regolare/inserire niente.....

ti basta rinominare la nuova entità così come si chiamava la precedente, che va a sostituire.

**Reply by faustozago** [reply]
Ottima soluzione, non ci avevo pensato.
Io ho risolto lasciando sia la vecchia entita' ,(anche se ho sostituito il dispositivo ), e aggiunto la nuova entita' che ha nome diverso , cosi nel pannello le vedo entrambe . 
 
Ora mi rimane un buco di circa una settimana (arrivo del nuovo dispositivo) dove non ho valori, quindi non posso neanche modificarli...
In questo caso cosa posso fare?

**Reply by KC-inDomus** [reply]
niente, se non andando a mettere le mani nel db
